changelogs/fragments/1653-cross_vc_clone_ssl_thumbprint.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmware_cross_vc_clone - Fix SSL Thumbprint validation to ignore if destination_vcenter_validate_certs is false
```

```suggestion
  - vmware_cross_vc_clone - Fix SSL Thumbprint validation to ignore if destination_vcenter_validate_certs is false
    (https://github.com/ansible-collections/community.vmware/issues/1433).
```

We usually reference the issue fixed or the PR in the changelog fragments.